example/App.vue

```diff
@@ -4,7 +4,13 @@
             {{ formData }}
         </pre>
         <div>
-            <super-form ref="form" :schema="exampleSchema" v-model="formData" :translations="translations" :render="renderFuncs"/>
+            <super-form 
+                ref="form" 
+                :schema="exampleSchema" 
+                v-model="formData" 
+                :translations="translations" 
+                :render="renderFuncs"
+                :renderOptions="renderOptionsFuncs"/>
```

Comment:
now that I look at it: this prop should be named _renderOption_ instead of _renderOption**s**_